tests/test_downloadermiddleware_httpcompression.py

```diff
@@ -26,10 +26,23 @@
 
 class HttpCompressionTest(TestCase):
 
+    def create_spider_mw(self
+                         ,compression_enabled=True
+                         ,compression_header=False):
+        crawler = get_crawler(Spider,
+                              {'COMPRESSION_ENABLED':
+                               compression_enabled,
+                              'COMPRESSION_KEEP_ENCODING_HEADERS':
```

Comment:
```suggestion
                               'COMPRESSION_KEEP_ENCODING_HEADERS':
```

Reply:
I was trying to hard to stick to the 80 width limit here.